### ISSUE TITLE
Check automatic person tagging in ch. 13

### ISSUE THREAD

**Issue by nathangibson** [usaybia/usaybia-data#49]
Hi @RobinSchmahl (and @lunadine FYI)
I've done some automated person name tagging in ch. 13 and merged it into your branch here: https://github.com/usaybia/usaybia-data/blob/tag-names-with-URIs-/-Robin-Schmahl-27th-May-2020/data/texts/tei/lhom-ar-13.xml. 

Just a note: I've used addName and forename elements inside persName to split the names into parts. We won't necessarily keep these but may help us for the moment be able to grab specific things.

Could you evaluate it for the following please? It doesn't have to be perfect, but I want to see how we could use this process to see how we could speed up our work.

1. Are most named people tagged? Are there any patterns for the names that were missed? (I haven't tried to tag any anonymous persons.) 
1. Do the persName tags usually include the entire name, not leaving out parts of the name or including things that are not part of the name?

Once we have some good results, we'll develop a system for you to tag several occurrences of a person's name at once. The question is, what is the best way to sort names so that you can reliably tell which people are the same (in cases where the name itself is enough to tell the identity)? I've split the names into 3 different lists here: https://github.com/usaybia/usaybia-tools/tree/master/tools/xquery/list-names-by-type-results

1. Shuhra: In some cases, IAU says  X المعروف ب. I've marked these as  "shuhra". Also if he uses Ibn X I've tagged these also as "shuhra", since the person can apparently go by this rather than the full genealogy. Anyone who has a shuhra I've put in this list, sorted alphabetically by shuhra.
1. Given names: Where I've been able to tag given names by the format of the name, this list organizes them alphabetically. 
1. Other names: All names where I couldn't detect a shuhra or given name I put in this list and sorted alphabetically by the entire list. 

So the question is, would lists like this help you see several instances of a name you could tag all at once as the same person? Do you think of better ways of sorting or grouping them?

You can mention your feedback just here in the issue. If you're looking at a file on Github and click the line number, you can copy that link to mention a specific line here. 

Thanks!

**Comment by nathangibson** [comment]
@RobinSchmahl I thought of another way to organize the list -- as a spreadsheet so you can sort by whichever part of the name you want to. I also added kunya. 
Here's the spreadsheet, what do you think? https://docs.google.com/spreadsheets/d/1QZtZpiLiVwrNUay6j7EI4qB-2kaqtTMqNZkng3C6cQc/edit#gid=0

Note: The ref might not always be exactly accurate, but it should generally be close.

**Comment by nathangibson** [comment]
@lunadine I would be happy for you to take a look at this as well, since you're quite familiar with how IAU uses person names.

**Comment by nathangibson** [comment]
Hi @RobinSchmahl Did you get a chance to look at this?

**Comment by RobinSchmahl** [comment]
@nathangibson 

I've just had look until chapter 13.3.2.1.
The system is far from perfect, however I believe using it as a base to go over the code manually afterwards would speed up the process greatly. I think that the system does tag a name correctly, when it recognizes it as such (with the exception of زيادة الله in line 193/194) - however it still does not recognize some names (see point 3). So far the issues I found are the following:

1. The system seems to tag the same persons within a paragraph to often. (we only want the same person tagged once within a paragraph). 

2. The system does not recognize suffixes indicating persons-. For instance: the nī (=IAU) in حدثني in line 320.

3. Some shorter names or parts of names or shuhras are not recognized. e.g.

- line: 197/198/341: اسحاق
- line 202/293: اسرائيلي
- line 260: عباس
- line 264: ابقراط 
- line 264/266/271: جالينوس
- line 297: عبيد الله المهدي (in the footnote)
- line 343: المهدي
- line 403: ابن الجزر

with regards to the spreadsheet: I think this will be helpful! So far I do not have a better idea for sorting them,( perhaps once we will start with the process of tagging several occurrences of one name at once).

**Comment by nathangibson** [comment]
Thanks @RobinSchmahl , this is good to know! 

> 
>     1. The system seems to tag the same persons within a paragraph to often. (we only want the same person tagged once within a paragraph).

This is actually not a problem. The main reason to tag a person only once per paragraph is to save effort when we're doing it manually.
> 
>     2. The system does not recognize suffixes indicating persons-. For instance: the nī (=IAU) in حدثني in line 320.

Hmm, yes, maybe I should try adding automatic tagging of `<rs>`. But I'm not sure if I'll be able to make it accurate enough to help.
> 
>     3. Some shorter names or parts of names or shuhras are not recognized. e.g.

OK, not surprising. There is a way we could try to capture these but it involves a totally different method. I'm debating whether it's worth it. 

I think you've checked far enough, thanks. I'll remove you from this issue and assign myself, since I need to make some tweaks.
